cli/Valet/Site.php

```diff
@@ -649,7 +649,7 @@ public function createPrivateKey(string $keyPath): void
     public function createSigningRequest(string $url, string $keyPath, string $csrPath, string $confPath): void
     {
         $this->cli->runAsUser(sprintf(
-            'openssl req -new -key "%s" -out "%s" -subj "/C=/ST=/O=/localityName=/commonName=%s/organizationalUnitName=/emailAddress=%s%s/" -config "%s"',
+            'openssl req -new -key "%s" -out "%s" -subj "/C=/ST=/O=/localityName=/commonName=%s/organizationalUnitName=/emailAddress=%s%s/"',
             $keyPath, $csrPath, $url, $url, '@laravel.valet', $confPath
```

Comment:
Should `, $confPath` be removed as well, since the previous line removed the `%s` which referenced it?

Reply:
Thanks for this catch, fixed